test/e2e/reconciler/namespace/controller_test.go

```diff
@@ -230,3 +316,14 @@ func expectNamespaces(ctx context.Context, t *testing.T, client kubernetes.Inter
 		}, 30*time.Second)
 	}, nil
 }
+
+func patchNamespaceLabel(label, value string) []byte {
```

Comment:
Maybe just make this public in the target package so it can be reused?

Reply:
I've made the function in the target package and removed that one.